appleid.md

```diff
@@ -0,0 +1,27 @@
+# Apple ID
+Apple IDs are a requirement for Macbook support. 
+## Installation
+You will receive an invitation via email containing 
+* Your managed Apple ID eg: user@studio24.net.
+* Your temporary password
+
+
+* Click the link in the email or visit the URL [Apple Business Manager](https://business.apple.com)
+* Enter your Apple ID
+* Enter your temporary password
+* Add a telephone number, this is used for verification when logging in
+* Verify your telehone
+* Create a new password
+* Store the new password in your private vault in 1Password
+* Verify your email
+
+Once the Apple ID is setup, add it to your Macbook
+* Open System Preferences
+* Open Internet Accounts
+* Select iCloud
+* Enter your Apple ID, then select Next
+* Enter your password, then select Next
+* Enter the verification code
+* Enter your Mac login password
+* Select the following apps for iCloud to sync Notes
```

Comment:
What does this mean? We don't really use Notes across the team so I wouldn't suggest it. Presumably nothing is required so people can enable any iCloud services they wish to use?

Reply:
Sorry meant to remove the word Notes. Adjusted the text.